cli/src/main/java/com/box/l10n/mojito/cli/command/PushCommand.java

```
@@ -146,4 +164,28 @@ public void execute() throws CommandException {
 
         consoleWriter.fg(Ansi.Color.GREEN).newLine().a("Finished").println(2);
     }
+
+    /**
+     * Get fileMatch list contains git changes against git tree HEAD
+     * @param sourceFileMatches
+     * @return
+     * @throws CommandException
+     */
+    private ArrayList<FileMatch> getGitChangeFiles(ArrayList<FileMatch> sourceFileMatches) throws CommandException {
+        GitRepository gitRepository = new GitRepository();
+        gitRepository.init(commandDirectories.getSourceDirectoryPath().toString());
+        Git git = new Git(gitRepository.jgitRepository);
+        String gitPath = gitRepository.getDirectory().getPath().replace(".git", "");
+        Set<String> gitDiffFiles = new HashSet<>();
+        try {
+            List<DiffEntry> diffEntries = git.diff().setOldTree(gitRepository.getHeadTree()).call();
+            gitDiffFiles.addAll(diffEntries.stream().map(DiffEntry::getNewPath).collect(Collectors.toSet()));
+        } catch (IOException | GitAPIException e) {
+            logger.debug(e.getMessage());
```

when is this happening? is it fine to silently fail?